README.md

```diff
@@ -15,8 +15,16 @@ Pull AEM Platform BuildEnv Docker image from Docker Hub:
 
 Or alternatively, you can create the Docker image:
 
+    Install Hashicorp packer tool for your OS:
```

Comment:
Just minor, please align the commands so it's indented the same to the command below in this section.
And just 'comment out' the description for consistency with other READMEs, e.g. `# Install Hashicorp ...` and `# Install Ruby bundler ...`